cartesian_impedance_msgs/msg/SetCartesianImpedance.msg

```diff
@@ -1,10 +1,13 @@
 # Here we combine all the parameters message definitions into a single message
 # that needs to be send to the controller in order to adjust the Cartesian Impedance Parameters
 
-string               tool
-CartesianStiffness   stiffness
-CartesianDamping     damping
-MaxCartesianVelocity max_cart_vel
-MaxControlForce      max_ctrl_force
-MaxPathDeviation     max_path_deviation
-NullSpace            null_space_params
+string                 tool
+CartesianStiffness     stiffness
+#use this if the damping is in Ns/m, Ns/rad
```

Comment:
Do you mean Cartesian Damping type? 

For now the only difference is in the naming, and is up to the user to decide which one to choose. In fact there is not even difference of the variable definition. 

Do you have any suggestions how we can differentiate this better, or we could just leave one msg type called damping and let the user either input ratio or Ns/m for the damping parameters. Having two types of damping msgs can be confusing and prone to errors